src/commands/interface-manager/MatrixHelpRenderer.tsx

```diff
@@ -63,13 +63,27 @@ export function renderCommandHelp(command: InterfaceCommand<BaseFunction>): stri
     ].join(' ');
 }
 
-export async function renderHelp(client: MatrixSendClient, commandRoomId: string, event: any, result: CommandResult<InterfaceCommand<BaseFunction>[], CommandError>): Promise<void> {
-    const commands = result.ok;
-    let text = ''
-    for (const command of commands) {
-        text += `${renderCommandHelp(command)}\n`;
+function renderTableHelp(table: CommandTable): DocumentNode {
+    // FIXME: is it possible to force case of table names?
```

Comment:
Well meow, what do you mean?

Reply:
Uhhhhhh I have to look that up Oo. I didnt even remember writing it. I think that fixme can go away.

Reply:
It's possible that it's copy pasted from somewhere else, ie it would be my FIXME that's gotten lost

Reply:
I think that may have happened. In the end this is mostly copied from the normal bot command handler.